src/main/java/org/folio/circulation/resources/AbstractCirculationRulesEngineResource.java

```diff
@@ -38,6 +38,7 @@
   public static final String PATRON_TYPE_ID_NAME = "patron_type_id";
   public static final String SHELVING_LOCATION_ID_NAME = "shelving_location_id";
   public static final String LOAN_TYPE_ID_NAME = "loan_type_id";
+  public static final String INSTITUTION_LOCATION_ID_NAME = "institution_location_id";
```

Comment:
Can you use the new naming scheme everywhere?
Can you rename
SHELVING_LOCATION_ID_NAME to LOCATION_ID_NAME,
"shelving_location_id" to "location_id",
INSTITUTION_LOCATION_ID_NAME to INSTITUTION_ID_NAME,
"institution_location_id" to "institution_id"?

Reply:
Yes, I will do it.

Reply:
@julianladisch  @VRohach Is `shelving_location_id` the name of a current parameter to the circulation rules?

If it is, renaming it is a breaking compatibility change.

Reply:
@marcjohnson-kint I just want to describe naming changes.

We added a mechanism for Institutions.
At first, classes for processing rules were renamed by this way: 
_CampusLocation.java -> Institution.java
BranchLocation.java -> Campus.java
CollectionLocation.java -> Library.java
ShelvingLocation.java -> Location.java_

Moreover, the new parameter "**institution_location_id**" was added to the request with these parameters:
"item_type_id=%s&loan_type_id=%s&patron_type_id=%s&shelving_location_id=%s".

Then, as far as I understood Julian`s comment, we need to rename "**shelving_location_id**" to "**location_id**" and "**institution_location_id**" to "**institution_id**" for naming schema consistency. 

So, @julianladisch  could you please provide your comments about Mark suggestion?

Reply:
Yes, renaming `shelving_location_id` to `location_id` is a breaking compatibility change.
This is to adopt the new naming scheme of mod-inventory-storage where shelf-location.raml has been replaced by location.raml and locationunit.raml: https://dev.folio.org/reference/api/#mod-inventory-storage

The breaking compatibility change without some transition time where we support both names is justified by the small number of uses: https://github.com/search?q=org%3Afolio-org+shelving_location_id&type=Code

To reflect the breaking compatibility change we need to bump the circulation version to 8: https://github.com/folio-org/mod-circulation/blob/master/descriptors/ModuleDescriptor-template.json#L43

Reply:
We also need pull requests for every client module in the platforms raising and approving before this can be merged.

I think this will likely mean that this should not be attempted till after the Q3.2 deadline next week, but I need to review that situation more broadly.

Reply:
Yes.
We should also consider splitting the many "circulation" endpoints in the ModuleDescriptor into smaller sections, for example a "circulation-rules" section for the /circulation/rules* APIs.